example/shared/src/main/scala/scalaz/example/TheseUsage.scala

```diff
@@ -0,0 +1,49 @@
+package scalaz
+package example
+
+import data._, typeclass._
+
+object TheseUsage extends App with Instances {
```

Comment:
👍 on the examples. Maybe nice to add one for `swap` or any other common methods.